### DISCUSSION TITLE
New device from IOS 15.1?

### DISCUSSION THREAD

**Discussion by aLAN-LDZ** [HomeSpan/HomeSpan#176] [Ideas]
Hi

Are there any plans to introduce new homekit devices that apple added in ios 15.1?

**Comment by HomeSpan** [comment]
I am currently adding Television Services - should be available in a week or so.  Are there others you know of (note that Apple has not updated its HAP documentation so all new services are a bit experimental)?  

I will also be adding in a macro so that you can easily add in any new Services and Characteristics (both Apple and non-Apple)

**Reply by aLAN-LDZ** [reply]
Previously, it was possible to add a temperature and humidity sensor itself?

https://www.youtube.com/watch?v=Nj9dgu4RCmE

I want to make a temperature and humidity sensor at home - a weather station

**Comment by HomeSpan** [comment]
I'm current running iOS 15.0.2 (latest release).  Temp Sensor is still there - I just tried out the HomeSpan Tutorial 12 (Service Loops) and the Temp Sensor works as expected.   The video above discusses new automations that use existing Temp and Humidity Services that will be available in iOS 15.1.  This is all within the Home App on your iPhone - I do not believe any changes will be required in HomeSpan but will test things out once the 15.1 is released.
